### PR TITLE
Fix cowswap name & add Solana DEXs to dex.info

### PR DESCRIPTION
**Pull request by hildobby**
All project names are snake_case except some of cowswap (some of cowswap trades are already snake_case), so here is a fix for that. Also, fyi, you can get more info by joining the project column with `dex.info` (ie dex type [aggregator, direct], properly capitalised name and twitter handle). More columns can always be added to this metadata table btw if you have some ideas
cc @bh2smith  @gentrexha 

To be merged after https://github.com/duneanalytics/spellbook/pull/5070

### CODE REVIEW

File: models/dex/dex_info.sql [hunk 1]
```diff
@@ -72,7 +72,6 @@ FROM (VALUES
     , ('0x API', '0x API', 'Aggregator', '0xProject') -- To Fix, should be just zeroex
     , ('paraswap', 'ParaSwap', 'Aggregator', 'paraswap')
     , ('cow_protocol', 'CoW Swap', 'Aggregator', 'CoWSwap')
```
Comment (bh2smith) [inline models/dex/dex_info.sql:74]: This name could also be changed to CoWSwap if need be.

**Review by bh2smith — APPROVED** [review]
Seems legit.

File: models/dex/dex_info.sql (hunk 1 above)
Comment (hildobby) [reply models/dex/dex_info.sql:74]: ah is `CoWSwap` the proper name and not `CoW Swap`? i want to make sure it's as accurate as possible

Comment (bh2smith) [reply models/dex/dex_info.sql:74]: I guess technically it's two words so can be left as is. Just thought you were aiming to remove spaces etc

Comment (bh2smith) [reply models/dex/dex_info.sql:74]: But if you look two lines above, I still see "0x API" as the project field.

**Comment by gentrexha** [comment]
fw: @olgafetisova

File: models/dex/dex_info.sql [hunk 2]
```diff
@@ -2,7 +2,7 @@
         tags = [ 'static'],
         schema='dex',
         alias = 'info',
-        post_hook='{{ expose_spells(\'["ethereum", "polygon", "bnb", "avalanche_c", "gnosis", "fantom", "optimism", "arbitrum", "celo", "base"]\',
+        post_hook='{{ expose_spells(\'["ethereum", "polygon", "bnb", "avalanche_c", "gnosis", "fantom", "optimism", "arbitrum", "celo", "base", "soalana"]\',
```
Comment (jeff-dude) [inline models/dex/dex_info.sql:5]:
```suggestion
        post_hook='{{ expose_spells(\'["ethereum", "polygon", "bnb", "avalanche_c", "gnosis", "fantom", "optimism", "arbitrum", "celo", "base", "solana"]\',
```

**Review by jeff-dude — CHANGES_REQUESTED** [review]

File: models/dex/dex_info.sql (hunk 1 above)
Comment (hildobby) [reply models/dex/dex_info.sql:74]: Yeah project field should all me snake_case but some like 0x remain to be fixed, will do that in other PRs. The name column in .info should be the properly capitalised & spaced name just to use for visualisations

**Review by jeff-dude — APPROVED** [review]
works for me

tagging @olgafetisova @harisang one more time to see if this impacts any usage of their spells on the dune app

**Comment by jeff-dude** [comment]
@hildobby merge conflicts have shown up here, after modifications to the spell in other PRs. can you clean this up again, then we can proceed? i took a look to resolve, but seems like some choices to make & didn't want to influence

**Review by bh2smith — APPROVED** [review]
Seems legit.

**Comment by harisang** [comment]
Sorry, missed this PR before. Looks fine!

Comment: since #5070 was closed, I would rename and update the description of this PR as it also includes changes unrelated to CoW Swap.

**Review by jeff-dude — APPROVED** [review]

**Pull request merged**